app/javascript/lib/scp-api-metrics.js

```diff
@@ -144,11 +146,38 @@ export function logFilterSearch(facet, terms) {
   )
 }
 
+/**
+ * Logs time between user action and its last interactive effect.
+ *
+ * This provides a higher-level view of timing information also available
+ * in more granular events, but in manner that's easier to find and less
+ * implementation-specific.  See also: logPlot in javascripts/application.js.
+ */
+export function logUserAction(lastEvent, perfTime) {
+  var isScatter = lastEvent === 'plot:scatter';
+  var pageName = window.SCP.analyticsPageName;
+  var isStudyOverview = pageName === 'site-study';
+
+  // Consider using this construct more widely
+  var isFullyInteractive = typeof window.SCP.fullyInteractive !== 'undefined';
```

Comment:
what does this mean?

Reply:
I just added a comment in a2f157e saying it means the "action's main effects are shown and ready for input".  That's about as precise as I can make it while keeping the definition flexible enough to handle our varied actions (page views, searches, etc.).

Reply:
Thanks for addressing this in #718.